schema_based_on_diagram.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE patients (
+id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+name VARCHAR(50), 
+date_of_birth DATE
+);
+
+CREATE TABLE treatments (
+id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+type VARCHAR(50),
+name VARCHAR(50)
+);
+
+CREATE TABLE medical_histories (
+id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+admitted_at TIMESTAMP,
+patient_id INT REFERENCES patients(id),
+status VARCHAR(50)
+);

```

Comment:
- [ ] As the previous reviewer mentioned, Please add a table that represents the many-to-many relationship between `medical_histories` and `treatment` and you can call it something like `treatments_histories`. See this [link ](https://hasura.io/learn/database/postgresql/core-concepts/6-postgresql-relationships/#:~:text=have%20multiple%20articles.-,Many%2Dto%2DMany,-A%20many%20to)to understand more about how to address many-to-many relationships.